includes/filters.php

```diff
@@ -65,4 +65,102 @@ function give_set_settings_with_disable_prefix( $old_settings, $settings ) {
 		update_option( 'give_settings', $settings );
 	}
 }
+
 add_action( 'update_option_give_settings', 'give_set_settings_with_disable_prefix', 10, 2 );
+
+/**
+ * Check spam through Akismet.
+ *
+ * @since 1.8.15
```

Comment:
@jaydeeprami This issue shifted from `1.8.15` to `1.8.14`. Can you update it in this PR too?

Reply:
@ravinderk Sure, I will do.